File: python/problem-3/FilterAndSquare.py
def filter_and_square(numbers):
    result = []
    for num in numbers:
        if num > 0:
            square = num ** 2
            result.append(square)
    return result
result = filter_and_Square([-2, 0, 3, -1, 4, 5])
print(result)

File: javascript/problem-3/FilterAndSquare.js
function squareAndFilter(numbers) {
    const result = []

    for (let number of numbers) {
        if (number > 0) {
            const square = number ** 2
            result.push(square)
        }
    } return result

}

result = squareAndFilter([-2, 0, 3, -1, 4, 5])

console.log(result)